app/dashboard/views.py

```diff
@@ -262,14 +272,26 @@ def send_tip_2(request):
         }
         params = json.loads(request.body)
 
-        username = username or params['username']
-        access_token = request.session.get('access_token')
-        emails = get_github_emails(access_token) if access_token else [primary_email]
+        to_username = params['username'].lstrip('@')
+        try:
+            to_profile = Profile.objects.get(handle__iexact=to_username)
+            if to_profile.github_access_token:
+                to_emails = get_github_emails(to_profile.github_access_token)
```

Comment:
is it worth checking `to_profile.email` in this block of code?  as opposed to below

Reply:
👍